### PR TITLE
Fix missing old activities on task activity page and user popup in global activities [SCI-3285]

### PR DESCRIPTION
**Pull request by okriuchykhin**
Jira ticket: [SCI-3285](https://biosistemika.atlassian.net/browse/SCI-3285)

### What was done
Fixed missing old activities on task activity page and user popup in global activities

### CODE REVIEW

File: app/helpers/global_activities_helper.rb [hunk 1]
```diff
@@ -31,7 +33,7 @@ def generate_link(message_item, activity)
 
     case obj
     when User
-      return popover_for_user_name(obj, team, false, true)
+      return "[@#{obj.full_name}~#{obj.id.base62_encode}]"
```
Comment (radiokills) [inline app/helpers/global_activities_helper.rb]: I'm slightly triggered :)

Comment (okriuchykhin) [reply app/helpers/global_activities_helper.rb]: What's wrong here?

Comment (radiokills) [reply app/helpers/global_activities_helper.rb]: Sorry, didn't want to 'request changes'. Just 'base62' triggers me a bit :)

Comment (radiokills) [reply app/helpers/global_activities_helper.rb]: Nothing wrong.

**Pull request merged**